CMakeLists.txt

```diff
@@ -114,12 +114,12 @@ configure_file(./omrversionstrings.CMakeTemplate.h omrversionstrings.h)
 ###
 include(OmrDDRSupport)
 
-if(NOT DEFINED OMR_DDR_SET)
-	set(OMR_DDR_SET "omr")
-	make_ddr_set(${OMR_DDR_SET})
+make_ddr_set(omrddr)
+# Note  this is a temporary workarround for consumers which still use teh OMR_DDR_SET mechanism
```

Comment:
My comment relates to this (not outdated) commit.

Reply:
removed, since the only one this would help is openj9, which does not have fully functional ddr support from cmake anyway